dal/src/dal/taskDAL.ts

```
@@ -0,0 +1,138 @@
+        }
+    ): Promise<Task> {
+        try {
+            const [result] = await pool.query<ResultSetHeader>(
+                `INSERT INTO tasks
+                 SET name       = ?,
+                     project_id = ?,
+                     created_at = NOW(),
+                     updated_at = NOW()`,
+                [name, projectId]
+            );
+            return this.getTaskById({id: result.insertId});
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)